### ISSUE TITLE
[cvpaper] CVPR2019 #885 Deep Transfer Learning for Multiple Class Novelty Detection

### ISSUE THREAD

**Issue by kazuwjnlab** [kazuwjnlab/cvpaper#848]
## \#885 [Deep Transfer Learning for Multiple Class Novelty Detection](http://openaccess.thecvf.com/content_CVPR_2019/papers/Perera_Deep_Transfer_Learning_for_Multiple_Class_Novelty_Detection_CVPR_2019_paper.pdf)
Pramuditha Perera, Vishal M. Patel
### どんな論文か？
新規性検知(novelty detection)をend-to-endで行うための新たな手法を提示した論文。
### 新規性
①損失関数としてcross-entropy lossに加え, noveltyかどうかを評価するためのmembership lossを定義し使用したこと。②外れ値として無関係な画像データ(out-of-distribution images)を学習に利用したこと。これによりinlierとoutlierを区別するためのフィルター(global negative filter)の学習を目指した
### 結果
データセットをknown classとnovel classに分割してnovel classを検出させる実験を，既存の4データセットに対して行った(out-of-distribution imagesにはPlaces365やILSVRC12の画像を使用)。すべてのデータセットについてVGG16, AlexNetによる検出能(AUC)は既存の手法を上回っていた
### その他（なぜ通ったか？等）

![thumb](https://drive.google.com/uc?export=view&id=1-iXdIht_xjgnxVuCzUXaudRe-oDgWc-Q)
novelty detection, loss function, transfer learning, fine-tuning,
